### PR TITLE
feat(tools/tokens): generate tokens as native css variables

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Generate native CSS variables

GitHub issue number or Jira issue URL: https://github.com/adeo/mozaic-design-system/issues/890

### CODE REVIEW

**Review by corentin-verquin-lm — APPROVED** [review]

**Review by Marine-Fabien — APPROVED** [review]

**Comment by tiloyi** [comment]
@all-contributors please add @corentin-verquin-lm  for review, example

**Comment by tiloyi** [comment]
@all-contributors please add @Marine-Fabien  for bug, review

File: packages/tokens/actions/actions.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+          fs.writeFileSync(scssFile, content)
+          console.log(chalk.bold.green(`✔︎ ${scssFile}`))
+        }
+      } catch (err) {
+        console.error(err)
+      }
+    },
+    undo: function (dictionary, config) {
+      // console.log('Cleaning cssvariables_to_scss action')
```
Comment (mohamedMok) [inline packages/tokens/actions/actions.js:28]: You can remove or uncomment this line

**Review by mohamedMok — APPROVED** [review]

File: packages/tokens/actions/actions.js (hunk 1 above)
Comment (tiloyi) [reply packages/tokens/actions/actions.js]: Fixed

**Pull request merged**